pmd-java/src/test/resources/net/sourceforge/pmd/lang/java/rule/multithreading/xml/DoNotUseThreads.xml

```diff
@@ -29,32 +29,80 @@ extending threads is not allowed
 			]]>
         </code>
     </test-code>
-        <test-code>
+    <test-code>
         <description><![CDATA[
-implementing runnabel is not allowed
+        Implementing ExecutorService is not allowed
      ]]></description>
         <expected-problems>1</expected-problems>
         <code>
-    	<![CDATA[
-			public class OtherThread implements Runnable {}
-			]]>
+        <![CDATA[
+            import java.util.concurrent.ExecutorService;
+
+            public class PMDTestExecutorService implements ExecutorService {}
+         ]]>
         </code>
     </test-code>
-        <test-code>
+    <test-code>
         <description><![CDATA[
-Use of runnable is also not allowed
+        Extending AbstractExecutorService is not allowed
      ]]></description>
+        <expected-problems>1</expected-problems>
+        <code>
+            <![CDATA[
+            import java.util.concurrent.*;
+
+            public class PMDTestExecutorService extends AbstractExecutorService {}
+         ]]>
+        </code>
+    </test-code>
+    <test-code>
+        <description><![CDATA[
+        Using ExecutorService is not allowed
```

Comment:
FYI - the CDATA sections are here unnecessary, just write it without:

`<description>Using ExecutorService is not allowed</description>`